fairscale/experimental/nn/offload.py

```diff
@@ -0,0 +1,447 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
```

Comment:
here you can add a top level docstring for the file if you like.

Reply:
I am not sure about that yet. Let me punt on that for now.